src/backend/vulkan/src/device.rs

```diff
@@ -1156,6 +1172,36 @@ impl d::Device<B> for super::Device {
         }
     }
 
+    #[cfg(feature = "use-openxr")]
+    unsafe fn create_image_from_openxr_raw_image(
```

Comment:
Sounds good. I guess the feature gate can also be removed?
This is part of `pub trait Device<B>`

Reply:
yeah, I think adding trait methods behind a cfg flag is an anti-pattern. Since features are additive, if one crate uses X and implements its trait T, but another crate uses X with feature F, then this first implementation will fail at compile time, since an extra method would be unimplemented.